### DISCUSSION TITLE
Failed to compile

### DISCUSSION THREAD

**Discussion by elhadjcire224** [vercel/community#4943] [Help]
### Summary

![Capture d’écran du 2023-11-23 13-39-06](https://github.com/vercel/community/assets/85805093/f65a28b8-3f05-4f15-86bb-d6287f140892)

### Steps to Reproduce

pnpm build

**Comment by amyegan** [comment]
Hey @elhadjcire224! Do you get the same error if you use [Vercel CLI](https://vercel.com/docs/cli) to run [`vercel build`](https://vercel.com/docs/cli/build#) on your local copy of the repo? If so, you can add the [`--debug`](https://vercel.com/docs/cli/global-options#debug) option to log additional details.
